src/BinomialQueue/Internals.hs

```diff
@@ -666,6 +661,7 @@ foldlU f z (MinQueue ts) = foldl_ f z ts
 -- @since 1.4.2
 foldlU' :: (b -> a -> b) -> b -> MinQueue a -> b
 foldlU' f z (MinQueue ts) = foldl'_ f z ts
+{-# INLINE foldlU' #-}
```

Comment:
Did you check that this improves things?

Reply:
That was actually a mistake, and won't change anything at all. I'll remove it.